src/utils/withAltContext.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 export default function withAltContext(flux) {
-  return function (Component) {
+  return function(Component) {
```

Comment:
Do we want to have a space for named functions too or just anonymous functions?

Like in ES6:
```js
myFunction(args) {
}
```

or 
```js
myFunction (args) {
}
```